src/bin/keycloakify/generateJavaStackFiles.ts

```diff
@@ -16,21 +18,19 @@ export type BuildOptionsLike = {
 }
 
 export function generateJavaStackFiles(params: {
-    version: string;
```

Comment:
looked like duplication to me, let me know if I was too eager

Reply:
No you weren't 👍🏻